OxfordBodleian/Juel-Jensen/BDLaethb3.xml

```diff
@@ -189,29 +758,284 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                               <!-- How can this be expressed: The parchment is slightly wavy around the margins.--></condition>
                             </supportDesc>
 
                             <layoutDesc>
                                 <layout columns="2" writtenLines="17">
-                                  <!--  <locus from="" to="294rb"/>-->
-                                    <note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#23r"/>.</note>
+                                    <locus from="4ra" to="4va"/>
+                                    <locus from="15ra" to="94vb"/>
+                                    <locus from="97ra" to="145vb"/>
+                                    <locus from="147ra" to="294rb"/>
+                                    <note corresp="#textarea #margin">This is the main layout of the text of the Gospels. 
+                                        Data on text area and margin dimensions taken from <locus target="#23r"/>.</note>
                                     <dimensions unit="mm" xml:id="textarea">
                                         <height>294</height>
                                         <width>287</width>
                                     </dimensions>
+                                    <note>The characters per line per column are 11–13. On <locus target="#95r"/> and <locus target="#145v"/>, 
+                                        which are written in a smaller script on the same layout, the characters per line per column are
+                                        16–18.</note>
                                     <dimensions type="margin" unit="mm" xml:id="margin">
                                         <dim type="top">60</dim>
                                         <dim type="bottom">88</dim>
                                         <dim type="right">64</dim>
                                         <dim type="left">29</dim>
                                         <dim type="intercolumn">26</dim>
                                     </dimensions>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.</ab>
+                                </layout>
+                                
+                                <layout columns="3" writtenLines="27">
+                                    <locus from="5ra" to="11rb"/>
+                                    <!--<note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#"/>.</note>-->
```

Comment:
This info has not been taken? It should be added on the to-do agenda

Reply:
already on the list with requests for further actions :)